std-bits/table/src/main/java/org/enso/table/expressions/ExpressionVisitorImpl.java

```diff
@@ -56,6 +53,84 @@ public int getColumn() {
     }
   }
 
+  private static Value wrapAsColumn(Value value, Function<Object, Value> makeConstantColumn) {
+    if (value.isNull()) {
+      return makeConstantColumn.apply(value);
+    }
+
+    var metaObject = value.getMetaObject();
+    return metaObject != null
+            && metaObject.isHostObject()
+            && metaObject.asHostObject() instanceof Class<?>
+        ? makeConstantColumn.apply(value)
+        : value;
+  }
+
+  public static class Method {
+    private final Value ensoMethod;
+    private final boolean isVariableArgumentMethod;
+    private final boolean isStaticMethod;
+    private final Value staticsType;
+    private final String name;
+
+    public Method(Value module, Value type, String name, boolean variableArgumentMethod) {
+      this.name = name;
+      var context = Context.getCurrent().getBindings("enso");
+      final Value staticsModule =
+          context.invokeMember("get_module", "Standard.Table.Expression_Statics");
+      this.staticsType = staticsModule.invokeMember("get_type", "Expression_Statics");
```

Comment:
Are we going to call this for every name resolution?
We should keep a final static for these.

Reply:
Done